pyspark/bigdl/util/engine.py

```diff
@@ -67,8 +67,9 @@ def __prepare_bigdl_env():
 
     def append_path(env_var_name, path):
         try:
-            print("Adding %s to %s" % (path, env_var_name))
-            os.environ[env_var_name] = path + ":" + os.environ[env_var_name]  # noqa
+            if path not in os.environ[env_var_name]:
```

Comment:
what if path is a substring of another in the env_var_name, e.g. your path is "/ABC/DEF", and the env_var_name is "/ABC/DEF/GHI:/DDD/XXX". The path is not in the env but your check will consider it in the env, right?

Reply:
What you doubt is reasonable. But in actual case, it is used to adding jar to SPARK_CLASSPATH, will this scenario happen?

Reply:
that should be fine. But the code could not tell that. Should we change the path to jar_path to make it more clearer?

Reply:
I'm fine for changing the name to `jar_path` as `append_path` is used for adding jars only.

Reply:
we can solve it by checking path split by ":" to avoid the substring flaw.